clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -115,7 +115,9 @@
     public static final String MAX_POLL_INTERVAL_MS_DOC = "The maximum delay between invocations of poll() when using " +
             "consumer group management. This places an upper bound on the amount of time that the consumer can be idle " +
             "before fetching more records. If poll() is not called before expiration of this timeout, then the consumer " +
-            "is considered failed and the group will rebalance in order to reassign the partitions to another member. ";
+            "is considered failed and the group will rebalance in order to reassign the partitions to another member. " +
+            "For static members, the expiration will be extended by one <code>session.timeout.ms</code> after reaching " +
```

Comment:
`by one` ? remove `one` ?

`will be extended` -- only if `session.timeout.ms` is larger, right?

Thinking about it more, it seems that `max.poll.interval` does not matter at all for static members, but only `session.timeout.ms` is relevant?

Reply:
If `max.poll.interval` is still relevant then this seems a little confusing.  

Assuming it is the expiration determined by `max.poll.interval` + `session.timeout.ms` or `max.poll.interval` + `max(session.timeout.ms - max.poll.interval, 0)`.

Reply:
Thanks for the comments. The actual behavior would be `max.poll.interval.ms` + `session.timeout.ms` for static members if the `poll` interval was too long. Let me update the comment and see if this makes sense.